src/Symfony/Bundle/TwigBundle/TwigBundle.php

```diff
@@ -37,6 +38,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new TwigLoaderPass());
         $container->addCompilerPass(new ExceptionListenerPass());
         $container->addCompilerPass(new RuntimeLoaderPass(), PassConfig::TYPE_BEFORE_REMOVING);
+        $container->addCompilerPass(new BundleViewPathPass(), PassConfig::TYPE_BEFORE_REMOVING);
```

Comment:
Should we use a priority value instead?

Reply:
I don't think so. Using a PassConfig type ensures that all compiler passes running during an earlier phase will do the job, disregarding the priority.
Setting a priority will mechanically limit the possibilities to use the feature.

Reply:
> I don't think so. Using a PassConfig type ensures that all compiler passes running during an earlier phase will do the job, disregarding the priority.

We need to be careful with that, since you are making use of a skill for which it is not intended.

> Setting a priority will mechanically limit the possibilities to use the feature.

I'm not sure it's a big deal. Setting a low priority (e.g. -16) should be enough to open a hole for third-party bundles destined to use this feature. Note that you would only need 1 priority position in your original issue to solve it.

Reply:
OK, done